src/main/kotlin/me/gserv/fabrikommander/commands/TpaCommand.kt

```diff
@@ -0,0 +1,32 @@
+package me.gserv.fabrikommander.commands
+
+import me.gserv.fabrikommander.data.TeleportRequest
+import me.gserv.fabrikommander.utils.Context
+import me.gserv.fabrikommander.utils.Dispatcher
+import me.gserv.fabrikommander.utils.aqua
+import me.gserv.fabrikommander.utils.plus
+import net.minecraft.command.argument.EntityArgumentType
+import net.minecraft.server.command.CommandManager.argument
+import net.minecraft.server.command.CommandManager.literal
+
+class TpaCommand(val dispatcher: Dispatcher) {
+    fun register() {
+        dispatcher.register(
+            literal("tpa").then(
+                argument("target", EntityArgumentType.player()).executes(this::tpaCommand)
+            )
+        )
+    }
+
+    fun tpaCommand(context: Context): Int {
+        val target = EntityArgumentType.getPlayer(context, "target")
+        val source = context.source.player
+        val request = TeleportRequest(source = source, target = target, inverted = false)
+        request.notifyTargetOfRequest()
+        context.source.sendFeedback(
+            aqua("Teleport request sent to ") + target.displayName,
```

Comment:
#  Cancellation button

I think having a [Cancel] button is nice to have in case you send a tpa request to the wrong person. I know you can do /tpcancel <player>, but buttons are always easier for users.
```suggestion
            aqua("Teleport request sent to ") + target.displayName + 
            reset(" [") + hover(
                click(
                    red("Cancel"),
                    ClickEvent(
                        ClickEvent.Action.RUN_COMMAND,
                        "/tpcancel " + target.entityName // There can be multiple active requests
                    )
                ),
                HoverEvent(
                    HoverEvent.Action.SHOW_TEXT,
                    red("Click here to cancel the request.")
                )
            ) + reset("]"),
```